README.md

```diff
@@ -109,6 +109,10 @@ where `MyPrivateKey.p12` is the private development key exported from the system
 The full path to the keychain can then be sent to the Appium system using the `keychainPath` desired capability,
 and the password sent through the `keychainPassword` capability.
 
+### CI
```

Comment:
I would rather provide a full sentence (e.g. Continuous Integration). Not all people understand shortcuts